env/launch.go

```diff
@@ -1,6 +1,6 @@
 package env
 
-var LaunchEnvBlacklist = []string{
+var LaunchEnvBlocklist = []string{
```

Comment:
I did a quick google search and it seems like most folks are going with `denylist` instead of `blocklist`. Might as well use the same terminology others are adopting.

E.g. https://github.com/rails/rails/issues/33677

Reply:
Go have gone with `blocklist` https://go.googlesource.com/go/+/608cdcaede1e7133dc994b5e8894272c2dce744b

I can see the argument in favor of `denyList` in systems where we'd ever need to list `blocks` but I don't think that's the case for us.

I'd rather make the change than not have the PR accepted, but I do think blocklist reads slightly better, and makes slightly more sense.

Reply:
Actually, both of those comparisons are slightly off, we're not describing a service that blocks/denies/allows generic things. We could be a little more specific in this case and use `include` and `exclude`

Reply:
agree! `include` and `exclude` seem like the best/clearest option.